File: Feitos em Java/2582.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);
        int test = leitor.nextInt();
	for(int i=0;i<test;i++){
            int num1 = leitor.nextInt();
            int num2 = leitor.nextInt();
            int sum=num1+num2;
            switch(sum){
                case 1:
                    System.out.println("P.Y.N.G.");
                    break;
                case 2:
                    System.out.println("DNSUEY!");
                    break;
                case 3:
                    System.out.println("SERVERS");
                    break;
                case 4:
                    System.out.println("HOST!");
                    break;
                case 5:
                    System.out.println("CRIPTONIZE");
                    break;
                case 6:
                    System.out.println("OFFLINE DAY");
                    break;
                case 7:
                    System.out.println("SALT");
                    break;
                case 8:
                    System.out.println("ANSWER!");
                    break;
                case 9:
                    System.out.println("RAR?");
                    break;
                case 10:
                    System.out.println("WIFI ANTENNAS");
                    break;
                default:
                    System.out.println("PROXYCITY");
                    break;
            }
	}
    }
}

File: Feitos em C++/2582.cpp
#include<stdio.h>
using namespace std;
int main() {
	int test,i;
	scanf("%d",&test);
	for(i=0;i<test;i++){
		int num1,num2;
		scanf("%d",&num1);
		scanf("%d",&num2);
		int sum=num1+num2;
		switch(sum){
			case 1:
				printf("P.Y.N.G.\n");
				break;
			case 2:
				printf("DNSUEY!\n");
				break;
			case 3:
				printf("SERVERS\n");
				break;
			case 4:
				printf("HOST!\n");
				break;
			case 5:
				printf("CRIPTONIZE\n");
				break;
			case 6:
				printf("OFFLINE DAY\n");
				break;
			case 7:
				printf("SALT\n");
				break;
			case 8:
				printf("ANSWER!\n");
				break;
			case 9:
				printf("RAR?\n");
				break;
			case 10:
				printf("WIFI ANTENNAS\n");
				break;
			default:
				printf("PROXYCITY\n");
				break;
		}
	}
	return 0;
}
